DEVELOPMENT.md

```diff
@@ -1,109 +1,87 @@
+- [Testing](#testing)
+  - [Running unit tests](#running-unit-tests)
+  - [Running Cypress](#running-cypress)
+  - [Running Firebase tests](#running-firebase-tests)
+- [Additional Angular CLI commands](#additional-angular-cli-commands)
+  - [Linting](#linting)
+  - [Code scaffolding](#code-scaffolding)
+  - [Further help](#further-help)
+
+## Running Buzz About
+
+### Firebase Emulators
+
+To run the Firebase emulators:
+```
+npm run firebase:emulator
 ```
 
- * When it asks for a username, enter your GitHub username.
- * When it asks for a password, paste in the personal access token (***not*** your GitHub password).
- * When it asks for your email, enter your GitHub email. 
-
-Some sticking-points:
-
-1. To paste text into PowerShell, right-click once.
-2. On github.com/settings/tokens, the personal access token has a space after it&mdash;be careful that you don't copy that! :upside_down_face:
-
-![github.com/settings/tokens with the extra space character highlighted][i:The gosh-darn whitespace]
-
-If you run into trouble, you can always fall back to typing the personal access code by hand.
-
-## Installing the Angular command-line interface
-
-Angular provides a CLI with commands for some common operations, like `ng build`, `ng serve`, `ng lint`, and `ng test`. (There's also `ng generate`, which lets you quickly make a new component or service without having to write out the boilerplate every time.)
+You should be able to access the Firebase Emulator Suite page at http://localhost:4000/
 
-In order to use the Angular CLI, you'll need to install the `@angular/cli` package globally:
+### Development Server
 
-```sh
-npm install -g @angular/cli
+To run the development server:
+```
+ng serve
 ```
 
-(If you're on a Unix system, you may need `sudo`.)
+Navigate to http://localhost:4200/. The app will automatically reload if you change any of the source files.
 
-## Setting up Firebase
 
-[Firebase][] is a set of cloud services provided by Google. We use a few of these services in Buzz About. Notably, we're using [Cloud Firestore][], a database that syncs between multiple clients in real time.
+## Testing
 
-When working on Buzz About, you'll want to make sure your school Google account gets added to our Firebase project. (Professor KK should be able to add you.)
+### Running unit tests
 
-### Installing the command-line interface
+To run the unit tests:
+```
+ng test
+```
 
-Like Angular, Firebase provides a set of command-line tools. You can get these by running
+### Running Cypress
 
-```sh
-npm install -g firebase-tools
+To build the app and run the Cypress end-to-end tests, start the Firebase emulators and run:
+```
+npm run e2e
 ```
 
-The first command that you want to run is probably `firebase login`. This will open a web browser prompting you to log in with your Google account.
+To open Cypress (so you can run the tests interactively), start both the Firebase emulators and the development server and run:
```

Comment:
Is it all right if we switch the order of these phrases to make the development server more prominent? (Otherwise, people might gloss over it and assume that `npm run cy:open` automatically starts the development server.)

(Citation: I did that :upside_down_face:)

```suggestion
To open Cypress (so you can run the tests interactively), start both the development server and the Firebase emulators and run:
```